components/datetime/src/neo_skeleton.rs

```diff
@@ -144,6 +144,87 @@ impl IntoOption<YearStyle> for YearStyle {
     }
 }
 
+/// A specification for how precisely to display the time of day.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "serde",
+    serde(from = "TimePrecisionSerde", into = "TimePrecisionSerde")
+)]
+#[non_exhaustive]
+pub enum TimePrecision {
```

Comment:
I'm not convinced by the `Exact` suffixes. For seconds, `SecondsFixed` would be better, and I think this can extend to `MinuteFixed` and `HourFixed` as well.

Reply:
I expected a comment like this on the PR and my response is that this will be an easy search-and-replace after we bikeshed the names.

Reply:
Bikeshed can take place in the proposal doc. See "Alternate Naming Schemes"

https://docs.google.com/document/d/1SkxoitlCFiQ_KGW3dmRk7lbumGd_N1rfYJLMlkiXvh4/edit?tab=t.0#heading=h.behu6pc0pmkk